src/config/ConfigParser.ts

```diff
@@ -116,6 +116,38 @@ export function configParse5(slice: Slice | null | undefined) {
     throw new Error('Invalid config');
 }
 
+export function configParse6(slice: Slice | null | undefined) {
+    if (!slice) {
+        // no param in mainnet for now, so throwing will cause crash of parseFullConfig()
+        return null
+    }
+    const mintNewPrice = slice.loadCoins();
+    const mintAddPrice = slice.loadCoins();
+    return {
+        mintNewPrice,
+        mintAddPrice
+    };
+}
+
+export function configParse7(slice: Slice | null | undefined) {
+    if (!slice) {
+        throw Error('Invalid config');
+    }
+    const extraCurrencies = slice.loadDict(Dictionary.Keys.BigUint(32), Dictionary.Values.BigVarUint(5));
+    return [...extraCurrencies].map(e => ({
+      id: e[0],
+      amount: e[1],
+    }))
+}
+
+export function configParseParamsIds(slice: Slice | null | undefined, unsigned: boolean) {
```

Comment:
In the schema these hashmaps look identical. Why do they use signed and unsigned keys in the code? Please provide some reference to where this is described.